packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -375,6 +376,9 @@ class WebServerDevice extends Device {
           ephemeral: false,
        );
 
+  static const String kWebServerDeviceId = 'web-server';
+  static bool showWebServerDevice = true;
```

Comment:
Is it better to assign false here?

Reply:
Not unless you've already updated all of our CI to provide this flag

Reply:
I see. Shall we create an issue and add it here as a comment?

Thanks a lot for adding the flag! :)

Reply:
Filled https://github.com/flutter/flutter/issues/71447